src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -214,6 +214,23 @@
           </div>
         </div>
 
+        <!-- Change Password -->
+        <div class="form-group row"
+             *ngIf="!isCurrentUser()">
```

Comment:
We'll need something similar to https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html#L106 I guess:

```suggestion
             *ngIf="!isCurrentUser() && !authStorageService.isSSO()">
```